packages/studio-ui/src/AppWithLazyLoading.tsx

```diff
@@ -18,7 +18,7 @@ export default function AppWithLazyLoading() {
 
   useEffect(() => {
     loadComponents();
```

Comment:
should we also add `void` before `loadComponents`? that must also be doing async work to load in the components

Reply:
I was also considering that.  The way that Oliver implemented this was he returned a list of void promises from the function, but I'm not entirely sure what the reasoning behind it was.

I think that the way the functions are implemented, though, are slightly different where we don't necessarily need the void on `loadComponents`.  For our `loadStyling` we needed to write the function very imperatively, so that we can properly await each dynamic import before beginning the next. This meant that the whole `loadStyling` function was async.  But, for `loadComponents` each dynamic import is independent of another, so Oliver called each dynamic import in a `map`, which as a whole didn't need to be awaited. So, `loadComponents` is not an "async" function itself.